src/hooks/useActiveAccount.tsx

```diff
@@ -120,43 +116,28 @@ export const ActiveAccountContextProvider = ({
     }
   }, [activeAccount?.account?.id, setStoredAccountId]);
 
-  // Clear selected account when
-  // we've detected that the userId has changed
-  useEffect(() => {
-    if (userId !== previousUserId) {
-      accountsResult.refetch();
-      setPreviousUserId(userId);
-    }
-  }, [previousUserId, userId, accountsResult]);
```

Comment:
This `useEffect` was actually not necessary (which made "previous user id" tracking unnecessary as well). Why: the entire react-query `QueryClient` gets cleared wiped when the user changes. When that happens, a refetch of the query is triggered automatically.

Reply:
> the entire react-query QueryClient gets cleared wiped when the user changes

That wasn't always the case. Are you certain it is setup that way?